gtsfm/utils/io.py

```diff
@@ -291,14 +347,28 @@ def write_images(gtsfm_data: GtsfmData, images: List[Image], save_dir: str) -> N
 
     num_imgs = gtsfm_data.number_images()
     # TODO: compute this (from keypoint data? or from track data?)
```

Comment:
remove the TODO if this is done?

Reply:
This is not a TODO I left or completed. I removed it though.

Reply:
(The TODO is indeed complete)